bin/smack.py

```diff
@@ -140,6 +198,14 @@ def arguments():
   verifier_group.add_argument('--smackd', action="store_true", default=False,
     help='generate JSON-format output for SMACKd')
 
+  svcomp_group = parser.add_argument_group('svcomp options')
+
+  svcomp_group.add_argument('--svcomp', action="store_true", default=False,
+    help='enter svcomp mode')
+
+  svcomp_group.add_argument('--error-witness', metavar='FILE', default=None, type=str, 
```

Comment:
I think that choosing an error trace format should be orthogonal to the input file, if at all possible. Anyhow, I think we should merge this pull request, and then refine this implementation, and also hopefully split it into modules. So let me do this merge first.